components/apimgt/org.wso2.carbon.apimgt.rest.api.store.v1/src/main/resources/store-api.yaml

```diff
@@ -3172,6 +3172,68 @@ paths:
           schema:
             $ref: '#/definitions/Error'
 
+  ######################################################
+  # The tenant resource APIs
+  ######################################################
+  '/tenants':
+    #--------------------------------------------------------
+    # Retrieve tenants by state
+    #--------------------------------------------------------
+    get:
+      security:
+        - OAuth2Security: []
+      x-wso2-curl: "curl -X GET \"https://localhost:9443/api/am/store/v1.0/tenants?state=active\" -H \"Authorization: Basic 3a7ecd76-4856-33ce-a703-e23e6cd24944\""
+      x-wso2-request: |
+        GET https://localhost:9443/api/am/store/v1.0/tenants/active-tenats?state=active
+        Authorization: Bearer ae4eae22-3f65-387b-a171-d37eaa366fa8
+      x-wso2-response: "{\"count\":3,\"list\":[{\"domain\":\"test.com\",\"status\":\"ACTIVE\"},{\"domain\":\"carbon.super\",\"status\":\"ACTIVE\"},{\"domain\":\"abc.com\",\"status\":\"ACTIVE\"}],\"pagination\":{\"offset\":0,\"limit\":25,\"total\":3,\"next\":null,\"previous\":null}}"
+      summary: |
+        Get get tenants by state
+      description: |
+        This operation is to get tenants by state
+      parameters:
+        - name: state
+          description: |
+            The state represents the current state of the tenant
+
+            Supported states are [ active, inactive]
+
+          in: query
+          type: string
+          required: false
+          default: 'active'
+          enum:
+            - active
+            - inactive
+        - $ref: '#/parameters/limit'
+        - $ref: '#/parameters/offset'
+      tags:
+        - active tenants
```

Comment:
We can simplify the tag as "Tenants"

Reply:
fixed https://github.com/wso2/carbon-apimgt/pull/6811/commits/a840167e827f482f100d25dadea5b4a46dc53be1